linux/main.go

```diff
@@ -61,7 +61,8 @@ func runServer(port int) (*server.Server, error) {
 		AuthMode:                "native",
 	}
 
-	db, err := server.NewStore(config, logger)
+	singleUser := len(sessionToken) > 0
```

Comment:
This makes no sense, it is going to be always true in this context.

Reply:
maybe it makes sense to keep consistency with everything else

Reply:
That was my reasoning, but I'm happy hardcoding `true` in this context as well :)